### ISSUE TITLE
need example DESCRIPTION file

### ISSUE THREAD

**Issue by vjcitn** [Bioconductor/BiocWorkshopSubmit#5]
README.md for this repo says
```
Workshop submitters should include the following files in their workshop:

A DESCRIPTION file with the following fields:

    Title: A title for the workshop
    Description: A short description of the workshop
    URL: A URL to the workshop repository from either GHCR or Docker Hub
```
I am having trouble with URL.  Is this for a container that may not yet exist?  What is the format of the URL field?

**Comment by LiNk-NY** [comment]
The container should exist if you are using the [BuildABiocWorkshop](https://github.com/Bioconductor/BuildABiocWorkshop) template. I will add a template URL directly below that section in the readme. Thanks

**Comment by LiNk-NY** [comment]
Resolved in 31d6bb794fd41a4af3564408486fbad5d27eb0ac
